src/pages/experiments/[experimentId]/data-processing/index.jsx

```diff
@@ -77,7 +77,7 @@ const DataProcessingPage = ({ experimentId, experimentData }) => {
   const pipelineHadErrors = pipelineErrors.includes(pipelineStatusKey);
   const pipelineNotFinished = pipelineRunning || pipelineHadErrors;
 
-  const completedSteps = pipelineStatus?.completedSteps;
```

Comment:
`pipelineStatus` is `null` if `backendStatus` is still loading. This evaluates to `null` which throws an error when `completedSteps` is used inside `StatusIndicator.jsx`.

Reply:
this is an error that's happening already in production and that you are fixing as part of the PR right? not directly related to the error boundary stuff

Reply:
Yes, that's correct.